File: tuts/012-transitgateway-gettingstarted/python/scenario_getting_started.py
# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
# SPDX-License-Identifier: Apache-2.0

import uuid
import boto3
from ec2_wrapper import TransitGatewayWrapper


def run_scenario():
    client = boto3.client("ec2")
    wrapper = TransitGatewayWrapper(client)
    print(f"Running TransitGateway getting started scenario...")
    # TODO: implement setup, interact, teardown
    print("Scenario complete.")


if __name__ == "__main__":
    run_scenario()

File: tuts/012-transitgateway-gettingstarted/ruby/scenario_getting_started.rb
# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
# SPDX-License-Identifier: Apache-2.0

require_relative 'ec2_wrapper'

def run_scenario
  client = Aws::Ec2::Client.new
  wrapper = Ec2Wrapper.new(client)
  puts "Running Ec2 getting started scenario..."
  # TODO: setup, interact, teardown
  puts "Scenario complete."
end

run_scenario if __FILE__ == $PROGRAM_NAME
